karma.conf.js

```diff
@@ -53,7 +53,7 @@ module.exports = function(config) {
     // - Safari (only Mac)
     // - PhantomJS
     // - IE (only Windows)
-    browsers: ['Firefox'],
+    browsers: ['PhantomJS'],
```

Comment:
I'd prefer to keep this as Firefox since Phantom can be extremely finicky (Doesn't support `Function.prototype.bind` among other things). We should probably remove it from package.json since we no longer need it.